README.md

```diff
@@ -52,6 +52,28 @@ class DonateForm(forms.Form):
                     MaxMoneyValidator(Money(500, 'EUR'))])
 ```
 
+And for multi-currency:
```

Comment:
It should be explained what 'with currency' means. All fields support currency - but this one can modify it